hca/dss/__init__.py

```diff
@@ -477,12 +510,17 @@ def _download_manifest_tasks(self, manifest, replica, num_retries, min_delay_sec
             for row in reader:
                 bundles[(row['bundle_uuid'], row['bundle_version'])].add(row['file_name'])
         for (bundle_uuid, bundle_version), data_files in bundles.items():
-            data_globs = tuple(glob_escape(file_name) for file_name in data_files if file_name)
+            if no_data:
+                data_filter = ('',)
+            else:
+                data_filter = tuple(glob_escape(file_name) for file_name in data_files if file_name)
+            metadata_filter = ('',) if no_metadata else ('*',)
```

Comment:
I'd either use if statement or if expression for both data and metadata. For symmetry.